rust/rope/src/rope.rs

```diff
@@ -490,7 +494,11 @@ impl Rope {
     }
 
     /// Returns a slice of the string from the byte range [`start`..`end`).
```

Comment:
this doc is now out of date. maybe just, 'Returns a new `Rope` with the contents of the provided range.`

Reply:
:+1: